sequencer/worker.go

```diff
@@ -55,6 +57,9 @@ func (w *Worker) AddTxTracker(ctx context.Context, tx *TxTracker) (replacedTx *T
 		// Unlock the worker to let execute other worker functions while creating the new AddrQueue
 		w.workerMutex.Unlock()
 
+		// Wait until all pending transactions are stored, so we can ensure getting the correct nonce and balance of the new AddrQueue
+		w.pendingTransactionsToStoreWg.Wait()
```

Comment:
can happen that a lot of transactions cotinue to being processed meanwhile and this never ends?

Reply:
There is a limit of txs that fit in a batch, so this will end before closing a batch.